modules/openapi-generator/src/main/resources/python/python-experimental/api_test.mustache

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
```

Comment:
Do we still need this new python-experimental file?
It looks the same as the file that we use for the python generator.
If I remember right, did we add this because we were originally going to update the api test examples in this PR then we downscoped to do them later?
If we don't need it can you remove it?

Reply:
Correct. Done

Reply:
Where was this done? I still see this file here.
Should we remove it?